src/server/dev/index.ts

```diff
@@ -42,6 +42,10 @@ class DevServer extends Dispose {
 
   private _onExit = (code: number) => {
     this.task = undefined;
+    const config = workspace.getConfiguration('flutter.commands');
+    if (config.get<boolean>('autoOpenDevLog', false) && this.outputChannel) {
+      this.outputChannel.hide();
+    }
```

Comment:
It maybe confused to auto hide the output buffer.

Reply:
Do you mean it is confusing to the user if the output closes automatically?
Because my idea of it was, we’re opening it automatically so then we should also close it automatically.

If you want we can add another setting `autoHideDevLog` I just felt it was overkill

Reply:
> Do you mean it is confusing to the user if the output closes automatically?

Yes. I think option `autoHideDevLog` would be better